backend/models/beneficiary_model.py

```diff
@@ -16,10 +16,10 @@
 
 
 class Beneficiary(Document):
-    first_name = StringField(max_length=500)
-    last_name = StringField(max_length=500)
-    phone = IntField(min_value=16, max_value=120)
-    landline = IntField(min_value=16, max_value=120)
+    first_name = StringField(max_length=500, required=True)
+    last_name = StringField(max_length=500, required=True)
+    phone = IntField(min_value=10000000, max_value=99999999)
+    landline = StringField(max_length=500)
```

Comment:
change phone to string, and add the max and min length 8, and regex to check for digits, I think also Vitalie have some changes in his PR to the beneficiary model

Reply:
I would suggest to keep phone as user entered it, we can create another field(phone_digits) if it will be needed for searching.

Reply:
if we have a prefix before the phone textbox in the frontend, they are allowed to enter just 8 digits.